requests_mv_integrations/request_mv_integration.py

```diff
@@ -0,0 +1,1774 @@
+        request_auth=None,
+        request_session=False,
+        cookie_payload=None,
+        build_request_curl=True,
+        allow_redirects=True,
+        verify=True,
+        stream=False,
+        verbose=False,
+        request_label=None
+    ):
+        """Request data from remote source with retries.
+
+        Args:
+            request_method: request_method for the new :class:`Request` object.
+            request_url: URL for the new :class:`Request` object.
+            request_params: (optional) Dictionary or bytes to be sent in the
+                query string for the :class:`Request`.
+            request_data: (optional) Dictionary, bytes, or file-like object to
+                send in the body of the :class:`Request`.
+            request_json: (optional) json data to send in the body of
+                the :class:`Request`.
+            request_retry: (optional) Retry configuration.
+            request_retry_func: (optional) Retry function, alternative
+                to request_retry_excps.
+            request_retry_excps: An exception or a tuple of exceptions
+                to catch.
+            request_headers: (optional) Dictionary of HTTP Headers to
+                send with the :class:`Request`.
+            request_auth: (optional) Auth tuple to enable Basic/Digest/Custom HTTP Auth.
+            allow_redirects: (optional) Boolean. Set to True if POST/PUT/DELETE
+                redirect following is allowed.
+            verify: (optional) whether the SSL cert will be verified. A
+                CA_BUNDLE path can also be provided. Defaults to ``True``.
+            stream: (optional) if ``False``, the response content will be
+                immediately downloaded.
+            verbose: (optional) Boolean, provide request and response details.
+            request_label:
+
+        Returns:
+            requests.Response: Data result if success or None if error.
+
+        Raises:
+            ServiceGatewayTimeoutError: Upon any timeout condition.
+            Exception: Upon error within this request_method.
+
+        Notes:
+            * tries: the maximum number of attempts. default: 1.
+            * delay: initial delay between attempts. default: 1.
+            * max_delay: the maximum value of delay. default: None (no limit).
+            * backoff: multiplier applied to delay between attempts.
+                default: 1 (no backoff).
+            * jitter: extra seconds added to delay between attempts.
+                default: 0.
+        """
+        self.logger.debug(
+            "Request: Start: {}".format(
+                request_label if request_label else ""
+            )
+        )
+
+        timeout = None
+
+        retry_tries = None
+        retry_delay = None
+        retry_backoff = 0
+        retry_jitter = 0
+        retry_max_delay = None
+
+        if not verify:
+            requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
```

Comment:
@zaquestion 
If placing this block of code within in `def init(...,verify,...)`, then we are assuming that all requests made by a specific mv-integration would have the same security needs, instead of limited by security needs by request.

I think that security should be disabled as need by request.